boards/arm/tiva/lm3s6965-ek/src/Makefile

```diff
@@ -46,4 +46,8 @@ ifeq ($(CONFIG_NX_LCDDRIVER),y)
 CSRCS += lm_oled.c
 endif
 
+ifeq ($(CONFIG_BOARD_LATE_INITIALIZE),y)
+  CSRCS += lm_bringup.c
+endif
+
```

Comment:
Normally in most existing configuration, board_app_initialize() and board_late_initialize() both call the bringup() functions.  These are two alternative ways to do the same thing:  Either the initialization is done on a separate initialization thread, or on the main thread of the application.  But, in either case, the initialization should be the same.

Reply:
done